app/gui2/src/util/reactivity.ts

```diff
@@ -156,3 +157,5 @@ export function syncSet<T>(target: Set<T>, newState: Set<T>) {
   for (const oldKey of target) if (!newState.has(oldKey)) target.delete(oldKey)
   for (const newKey of newState) if (!target.has(newKey)) target.add(newKey)
 }
+
+export type ToValue<T> = Parameters<typeof toValue<T>>[0]
```

Comment:
This is already defined in `placement.ts`, and I prefer the definition without using `Parameters`.